csg.js

```diff
@@ -6599,23 +6654,18 @@ for solid CAD anyway.
     };
 
     CAG.Side._fromFakePolygon = function(polygon) {
-        polygon.vertices.forEach(function(v) {
```

Comment:
_fromFakePolygon is only called from fromFakeCSG, which by definition has to be in that range of Z. Why check every single vertex? The Z is controlled internally.